Automattic/wp-calypso
# Fixed super props not being added in tracks when path is undefined

When the event properties don't have a property called 'path',  then we should fallback to the currentPath from Redux.

SInce the Redux path contains the actual path (not the route), we'll use getSiteFragment to guess if this is a site-specific page.
In this manner, every track event will be automatically populated with the blog_id and plan id automatically.


Related to #

## Proposed Changes

* Automatically pass the global information for every event (blog_id, blog_lang, site_plan). 

## Testing Instructions



* Use the calypso.live link and test a random event
* Make sure that the `blog_id`, `blog_lang` and `site_plan_id` parameters are automatically passed for paths that have a /:site in the URL.

## Pre-merge Checklist
